.github/workflows/ci.yaml

```diff
@@ -42,7 +42,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v5
         with:
-          go-version: 1.18
+          go-version: '1.21'
```

Comment:
migrating to 1.21 could be problematic. Has this been resolved? @cosmo0920 worked on this as well.

Reply:
Yes. The cherry-picked patch is already merged into 1.21 line: https://go-review.googlesource.com/c/go/+/549495
Moving to 1.22 is safer choice.